templates/taxi/driver_detail.html

```diff
@@ -22,5 +22,7 @@ <h4>Cars</h4>
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+  <a style="float: left" href="{% url "taxi:driver-update" pk=driver.id %}" class="btn btn-secondary">Update</a>
```

Comment:
The URL name `taxi:driver-update` might be confusing as it overlaps with the `DriverLicenseUpdateView`. Ensure that the URL name matches the intended view function or consider renaming for clarity.